src/js/utils/SetupUtil.js

```diff
@@ -28,6 +28,8 @@ const SetupUtil = {
         preparedData[key] = SetupUtil.getArrayFromHostsString(newFormData[key]);
       } else if (key === 'ssh_port') {
         preparedData[key] = parseInt(newFormData[key]);
+      } else if (key === 'nested_cluster_options') {
```

Comment:
It's a list of options for the cluster. That's the best name I could come up with. Do you have any other ideas?